packages/medusa/src/utils/validators/is-nullable.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  registerDecorator,
+  ValidationArguments,
+  ValidationOptions,
+} from "class-validator"
+import { IOptions } from "class-validator-jsonschema/src/options"
+
+export function IsNullable(validationOptions?: ValidationOptions) {
+  return function (object: Object, propertyName: string) {
+    registerDecorator({
+      name: "IsNullable",
+      target: object.constructor,
+      propertyName: propertyName,
+      options: validationOptions,
+      validator: {
+        validate(value: any, args: ValidationArguments) {
+          return value === null
+        },
+      },
+    })
+  }
+}
+
+export function IsNullableJSONSchemaConverter(
+  _meta,
+  _options: Partial<IOptions>
+) {
+  return { nullable: true }
```

Comment:
question: Is that always the case? In that case it might be worth setting it directly in the place it is used instead. Let me know

Reply:
Do you mean to replace `IsNullable: IsNullableJSONSchemaConverter,` by `IsNullable: () => return { nullable: true },` in [build-oas.ts](https://github.com/medusajs/medusa/blob/8381a5d5c50520e3c3ed613ce58a6d471ac3a9c2/packages/medusa/src/scripts/oas/build-oas.ts#L106)?

We could, but I would prefer to keep the JSONSchemaConverter method next to its decorator. I will allow to keep unit tests in the same place.